backend/src/main/java/ai/verta/modeldb/utils/JdbiUtil.java

```diff
@@ -0,0 +1,104 @@
+package ai.verta.modeldb.utils;
+
+import ai.verta.modeldb.common.CommonConstants;
+import ai.verta.modeldb.common.CommonDBUtil;
+import ai.verta.modeldb.common.CommonMessages;
+import ai.verta.modeldb.common.MssqlMigrationUtil;
+import ai.verta.modeldb.config.MDBConfig;
+import io.grpc.health.v1.HealthCheckResponse;
+import java.io.File;
+import java.sql.SQLException;
+import java.util.concurrent.Executor;
+import liquibase.exception.LiquibaseException;
+import liquibase.resource.FileSystemResourceAccessor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.hibernate.exception.JDBCConnectionException;
+
+public class JdbiUtil extends CommonDBUtil {
+  private static final Logger LOGGER = LogManager.getLogger(JdbiUtil.class);
+  private static JdbiUtil jdbiUtil;
+  private static String liquibaseRootFilePath;
+  private static boolean isReady = false;
+  private static MDBConfig config;
+
+  private JdbiUtil() {}
+
+  public static synchronized JdbiUtil getInstance(MDBConfig mdbConfig)
+      throws InterruptedException, SQLException {
+    if (jdbiUtil == null) {
+      jdbiUtil = new JdbiUtil();
+      config = mdbConfig;
+      liquibaseRootFilePath = "\\src\\main\\resources\\liquibase\\db-changelog-master.xml";
```

Comment:
curious why we're using windows file separators here. We should be able to use unix ones and have java do the right thing, correct?

Reply:
Java only taking this `"\\src\\main\\resources\\liquibase\\db-changelog-master.xml";` and it will automatically handled with unix and windows both.

Reply:
If I am setting `"/src/main/resources/liquibase/db-changelog-master.xml";` then the java process will fail and say about fix the path.

Reply:
that's ugly. so, windows separators work on linux, but not the other way around?

Reply:
@jkwatson-verta this `"\\src\\main\\resources\\liquibase\\db-changelog-master.xml";` is not specific to windows. the Path is actually `"\src\main\resources\liquibase\db-changelog-master.xml";` but In the java, `\` is pointing special character so we have to add one another `\` before that so the java is suggesting it so I don't think it is only related to windows.

Reply:
only windows uses the backslash as a path separator. :)